gcp/ignore_error_predicate.go

```diff
@@ -23,12 +24,21 @@ func isIgnorableError(notFoundErrors []string) plugin.ErrorPredicate {
 // shouldIgnoreErrorPluginDefault:: Plugin level default function to ignore a set errors for hydrate functions based on "ignore_error_codes" config argument
 func shouldIgnoreErrorPluginDefault() plugin.ErrorPredicateWithContext {
 	return func(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData, err error) bool {
-		if !hasIgnoredErrorCodes(d.Connection) {
-			return false
-		}
-
 		gcpConfig := GetConfig(d.Connection)
+
 		if gerr, ok := err.(*googleapi.Error); ok {
+			for _, pattern := range gcpConfig.IgnoreErrorMessages {
+				re := regexp.MustCompile(pattern)
+				result := re.FindString(types.ToString(gerr.Message))
+				if result != "" {
+					return true
+				}
+			}
+
+			if !hasIgnoredErrorCodes(d.Connection) {
```

Comment:
Do we need this block still?

Reply:
Removed this block of code as we do not need those.